File: javascript/006/main.js
/*
 * ProjectEuler 006
 * JavaScript / main.js
 * copyright (c) 2015 Susisu
 *
 * Node.js v0.12.0
 * `node main.js`
 */

function main() {
    var s = 0;
    var t = 0;
    for (var n = 1; n <= 100; n++) {
        s += n;
        t += n * n;
    }
    console.log(s * s - t);
}

main();

File: perl/006/main.pl
# ProjectEuler 006
# Perl / main.pl
# copyright (c) 2015 Susisu
#
# Perl 5.16.2
# `perl main.pl`

sub main {
    my $s = 0;
    my $t = 0;
    for (my $i = 1; $i <= 100; $i++) {
        $s += $i;
        $t += $i * $i;
    }
    print $s * $s - $t . "\n";
}

&main;
